cupy/cusparse.py

```diff
@@ -117,6 +117,36 @@ def cscsort(x):
         P.data.ptr, cusparse.CUSPARSE_INDEX_BASE_ZERO)
 
 
+def coosort(x):
+    handle = device.get_cusparse_handle()
+    m, n = x.shape
+    nnz = x.nnz
+
+    buffer_size = cusparse.xcoosort_bufferSizeExt(
+        handle, m, n, nnz, x.row.data.ptr, x.col.data.ptr)
+    buf = cupy.empty(buffer_size, 'b')
+    P = cupy.empty(nnz, 'i')
```

Comment:
Based on cusparse document, it must be `int` not be `size_t`  http://docs.nvidia.com/cuda/cusparse/index.html#cusparse-lt-t-gt-identity
http://docs.nvidia.com/cuda/cusparse/index.html#cusparse-lt-t-gt-coosort